examples/ts/btc/legacySafe/sweep.ts

```diff
@@ -0,0 +1,27 @@
+import { legacySafeConfig } from './config';
+import { AbstractUtxoCoin } from '@bitgo/abstract-utxo';
+import { unspents } from './fixtures/unspents';
+
+const destination = '2MyGxrhLC4kRfuVjLqCVYFtC7DchhgMCiNz';
+
+/*
+ * Usage: npx ts-node btc/legacySafe/sweep.ts
+ * */
+const sweepTransaction = async () => {
+  const coin = legacySafeConfig.sdk.coin(legacySafeConfig.coin) as AbstractUtxoCoin;
```

Comment:
would this `coin` field ever be anything other than `v1btc` ?

Reply:
nayeth, this is for v1 only.